README.md

```diff
@@ -91,7 +91,24 @@ make cluster/prepare/local
 ```
 
 ### Configuring Github OAuth
-Log in to RHSSO (see above) and click `Identity Providers` in the left sidebar. In the Github identity provider, find the Redirect URI and paste this URL into the Homepage URL and Authorization callback URL fields of your Github OAuth app.
+
+Follow [docs](https://docs.openshift.com/container-platform/4.1/authentication/identity_providers/configuring-github-identity-provider.html#identity-provider-registering-github_configuring-github-identity-provider) on how to register a new Github Oauth application and add the necessary authorization callback URL for your cluster as outlined
```

Comment:
@pmccarthy Would you mind adding something that shows this is only for OCP 4 clusters, not OSD clusters (because hive periodically resets the OAuth resource). In OSD, I believe the customer manages IDPs from OCM

Reply:
@david-martin Sure, this is done now